.github/workflows/ci.yml

```
@@ -158,60 +158,60 @@ jobs:
-          
+
       - name: Install dependencies
         run: npm ci
-        
+
       - name: Run security audit
         run: npm audit --audit-level=moderate
-        
+
       - name: Run CodeQL Analysis
         uses: github/codeql-action/init@v2
```

actionlint (1.7.7)

the runner of "github/codeql-action/init@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)